### PR TITLE
Update the XR documentation

### PR DESCRIPTION
**Pull request by m4gr3d**
This PR moves the Godot OpenXR documentation from the repo wiki.

_Bugsquad edit: Supersedes https://github.com/godotengine/godot-docs/pull/5480_

### CODE REVIEW

**Comment by m4gr3d** [comment]
@Calinou @mhilbrunner I've reopened the PR against the 3.4 branch and squashed the commits. It should be good for merging!

**Comment by m4gr3d** [comment]
@Calinou The checks are failing on a warning that seems unrelated to this PR. Any thoughts?

**Comment by mhilbrunner** [comment]
Ah, we need to sync upstream Godot docs classes with the docs repo for this branch. Nothing you need to fix!
This looks good to go, thanks, I'll poke your for a rebase as soon as the warning was fixed by us.

File: tutorials/vr/openxr/index.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Select the ``addons/godot_openxr/scenes/first_person_controller_vr.tscn`` subscene
+and add it to your scene. 
+Right-click the added node and select **Editable Children** to gain access
+to some of the nodes in this subscene:
+
+.. image:: img/editable_children.png
+
+.. note::
+
+    If using version 1.1.0 or earlier of the plugin, it is important to
```
Comment (BastiaanOlij) [inline tutorials/vr/openxr/index.rst:34]: As 1.1.1 has now been released we could remove this I think..

**Review by BastiaanOlij — APPROVED** [review]
Looks great to me

**Comment by m4gr3d** [comment]
@mhilbrunner Has the upstream issue been resolved?
We're planning on pushing a blog post about the latest XR features and we're planning to link to the updated documentation. What's the timeline once this is merged for it to be live on the site?

**Comment by mhilbrunner** [comment]
I just fixed the CI for 3.4. Can you rebase/push for a new build? Once I merged it should be up on RTD in an hour or so tops.

**Comment by m4gr3d** [comment]
> @m4gr3d I just fixed the CI for 3.4. Can you rebase/push for a new build? Once I merge it should be up on RTD in an hour or so tops.

Done!

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
@m4gr3d Merged, thank you! 🎉

**Pull request merged**